kanidmd/src/lib/core/https/mod.rs

```diff
@@ -1173,6 +1177,13 @@ pub fn create_https_server(
     qe_w_ref: &'static QueryServerWriteV1,
     qe_r_ref: &'static QueryServerReadV1,
 ) -> Result<(), ()> {
+    let (log_tx, log_rx) = unbounded::<TreeProcessor<KanidmEventTag>>();
+    tracing::subscriber::set_global_default(TreeSubscriber::pretty(log_tx))
+        .expect("Global subscriber already set, this is a bug");
+    let middleware = TreeMiddleware::with_stdout();
```

Comment:
Probably needs a better name than just middleware.

Reply:
^ This, given that other middlewares might happen in the future.